app/services/idv/response_set.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module Idv
+  class ResponseSet
```

Comment:
can we have a test suite for this class?

Reply:
Done. Thoroughly.